cnames_active.js

```diff
@@ -1919,6 +1920,7 @@ var cnames_active = {
   "ngspice": "42ua.github.io/ngspice.js",
   "ni": "js-ni.github.io/ni",
   "nic": "nic.github.io",
+  "Linadevv": "Linadevv.github.io"
```

Comment:
```suggestion
  "linadevv": "linadevv.github.io"
```
Subdomains are not case sensitive